lib/papertrail/http_client.rb

```diff
@@ -42,47 +42,27 @@ def get(path, params = {})
       if params.size > 0
         path = "#{path}?#{build_nested_query(params)}"
       end
-      attempts = 0
-      begin
+
+      attempt 3, 5.0 do
```

Comment:
I think it'd be easier to understand if the arguments to `attempt` were given as a hash (pretty sure we want to continue supporting Ruby 1.9 so it'd actually have to be a `Hash` and not kwargs 😕).